parkhangs-expressapp/controllers/parkController.js

```diff
@@ -24,6 +24,23 @@ const getParks = async (req, res) => {
 
 }
 
+// Adds a new event
+const addRating = async (req, res) => {
+  var parkId = req.body.parkId;
+  var rating = req.body.rating;
+    // console.log(newEvent.parkId);
+
+    try {
+        res.setHeader('Content-Type', 'application/json');
+        let inserted = await database.collection('events').update( { _id: parkId },{ $push: { "ratings": rating } });
```

Comment:
is there a reason you're directly communicating with the db rather than using Mongoose?

Reply:
Oh, that's just way the way I've been doing it always. I'll look into refactoring it. Thanks!

Reply:
no worries! I would take a look at the other controllers to see how we use mongoose, it allows us to avoid hard coding collection strings and stuff